### PR TITLE
fuchsia_ctl

### PR DESCRIPTION
**Pull request by dnfield**
Adds a package for managing Fuchsia devices on infra.

### CODE REVIEW

File: packages/fuchsia_ctl/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# fuchsia_ctl
+
+This package is used by Flutter CI systems to manag paving and testing Fuchsia
```
Comment (tvolkert) [inline packages/fuchsia_ctl/README.md]: manage

Comment (christopherfujino) [inline packages/fuchsia_ctl/README.md]: *manage (sp)

File: packages/fuchsia_ctl/bin/main.dart [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+  parser.addCommand('test')
+    ..addOption('pm-path', defaultsTo: './pm', help: 'The path to the pm executable.')
+    ..addOption('identity-file', defaultsTo: '.ssh/pkey', help: 'The key to use when SSHing.')
+    ..addOption('target', abbr: 't', help: 'The name of the target to pass to runtests.')
+    ..addMultiOption('far', abbr: 'f', help: 'The .far files to include for the test.');
+
+  final ArgResults results = parser.parse(args);
+
+  if (results.command == null) {
+    stderr.writeln('Unknown command, expeced one of: ${parser.commands.keys}');
```
Comment (tvolkert) [inline packages/fuchsia_ctl/bin/main.dart]: expected

File: packages/fuchsia_ctl/tool/build.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  exit -1;
+}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"
+
+cipd ensure -ensure-file $DIR/ensure_file -root $DIR
+
+pushd $DIR/..
+
+# rm -rf build
```
Comment (digiter) [inline packages/fuchsia_ctl/tool/build.sh]: Intended to keep this comment?

Comment (dnfield) [reply packages/fuchsia_ctl/tool/build.sh]: Replacing it with a check that the directory doesn't exist.  It was annoying to have it removed, but it should be cleaned before building.

File: packages/fuchsia_ctl/README.md (hunk 1 above)
Comment (dnfield) [reply packages/fuchsia_ctl/README.md]: Done

Comment (dnfield) [reply packages/fuchsia_ctl/README.md]: Done

File: packages/fuchsia_ctl/bin/main.dart (hunk 2 above)
Comment (dnfield) [reply packages/fuchsia_ctl/bin/main.dart]: Done

**Comment by dnfield** [comment]
Hmm.  Is this the right repository?  I don't want to publish this one to pub, although I suppose we could.  Right now it's set to publish_to none.

**Pull request merged**